chore(dockerfile): bump to JDK 17 (#265)
camunda/docker-camunda-bpm-platform: README.md

Our docker images are using the latest LTS OpenJDK version supported by
Camunda Platform. This currently means:

 - Camunda 7.20 or later will be based on OpenJDK 17.

>  - Camunda 7.12 - 7.19 is be based on OpenJDK 11.

Reviewer: 🔧 

```suggestion
 - Camunda 7.12 - 7.19 is based on OpenJDK 11.
```

 - All previous versions are based on OpenJDK 8.

While all the OpenJDK versions supported by Camunda will work for Tomcat and WildFly, we will not

> provide a ready to use image for them. (Camunda Run supports only JDK 17.)

Reviewer: 💬 Wildfly also needs a minimum Java 11. I'm not sure how to formulate this.

Author: 👍 True, I will make an adjustment.

Our docker images are using the latest LTS OpenJDK version supported by
Camunda Platform. This currently means:

 - Camunda 7.20 or later will be based on OpenJDK 17.

>    - Camunda 7.19 image for Camunda Run is supported only for JDK 17.

Reviewer: ❓ shouldn't this be 7.20 for Run?

```suggestion
   - Camunda 7.20 image for Camunda Run is supported only for JDK 17.
```

 - Camunda 7.12 - 7.19 is based on OpenJDK 11.
   - Camunda 7.19 image for WildFly is supported only for JDK 11 and JDK 17.
 - All previous versions are based on OpenJDK 8.

While all the OpenJDK versions supported by Camunda will work (without the exceptions specified above),

> we will not provide a ready to use image for them.

Reviewer: 🔧 Maybe sounds a bit better.

```suggestion
While all the OpenJDK versions supported by Camunda will work with the exceptions specified above,
we will not provide ready to use images for them.
```
